procurement_service/models/procurement_order.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2017 Alfredo de la Fuente - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo import models, api
+
+
+class ProcurementOrder(models.Model):
+    _inherit = 'procurement.order'
+
+    @api.multi
+    def _is_procurement_service(self):
+        return self.product_id._is_service_buy_make_to_order() or False
+
+    @api.model
```

Comment:
`_assign` is `@api.multi`. Please respect the method declaration.

Reply:
You are right, done